server/middleware/handleError.js

```diff
@@ -0,0 +1,10 @@
+// Middleware requires 4 parameters to be recognized by Express
+// eslint-disable-next-line no-unused-vars
+const handleError = (err, req, res, next) => {
+    const statusCode = err.statusCode || 500;
+    res.status(statusCode).json({ error: err.message });
+};
+
+module.exports = {
+    handleError
+};
```

Comment:
It is a pretty generic function, not just for making it JSON readable. Express' default error catching doesn't handle status codes predictably etc. It was also a good chokepoint for any customization and logging related to errors.

This is irrelevant though because all of the Express stuff is [already slated for the chopping blocking](https://app.asana.com/0/1203924324409025/1206058378518815/f) since it was originally put in place as part of an older design for the system. It didn't make sense to get rid of it before MVP but it is tracked as part of follow-up debt.